control/statesp.py

```diff
@@ -386,22 +386,98 @@ def horner(self, s):
         return array(resp)
 
     # Method for generating the frequency response of the system
-    def freqresp(self, omega):
-        """Evaluate the system's transfer func. at a list of ang. frequencies.
+    def freqresp(self, omega_s):
```

Comment:
Is there a reason to use `omega_s` here instead of `omega`?  In all other docstrings for `freqresp` (eg, `xferfcn.py`, `lti.py`) we just use `omega`.  Also, shows up as `omega` below.  I would leave as `omega` for consistency.